src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/iceberg/ApacheIcebergColumnSchema.java

```diff
@@ -3,22 +3,22 @@
 import org.apache.iceberg.types.Type;
 
 /** Wrapper class for Iceberg schema retrieved from channel. */
-public class ApacheIcebergColumnSchema {
+class ApacheIcebergColumnSchema {
```

Comment:
I dont think we need org name, we never used it any other places
```suggestion
class IcebergColumnSchema {
```